### ISSUE TITLE
Current loadout

### ISSUE THREAD

**Issue by raptor** [bitfighter/bitfighter#431]
_From [watusim...@bitfighter.org](https://code.google.com/u/105427273526970468779/) on June 01, 2014 05:26:29_

On one occasion, with large lag, I noticed that after updating my loadout, the loadout indicator did not get updated.  My loadout was correct, it was only the indicator that was wrong.

My theory is that when you change loadouts, the new loadout is sent via a packet that could be dropped.  If this is the case, we need to send new loadouts, or the signal that the new loadout has been accepted, via a s2c, so we can verify the packet was recieved.

_Original issue: http://code.google.com/p/bitfighter/issues/detail?id=431_

**Comment by raptor** [comment]
_From [buckyballreaction](https://code.google.com/u/buckyballreaction/) on July 09, 2015 20:23:54_

This is because a LoadoutMask is sent with pack/unpackUpdate and is droppable.

Perhaps it should be an s2c?

**Comment by eykamp** [comment]
Yes, it should be an s2c.  Much of what we transmit in these droppable packets should not be there.

**Comment by eykamp** [comment]
This will require a version increase; can't be done in 019x.
